Games/LLEGSB/script.js

```diff
@@ -0,0 +1,269 @@
+const app = new PIXI.Application({ backgroundColor: 2e0025 });
+document.body.appendChild(app.view);
+// default text style
+const style = new PIXI.TextStyle({
+  align: "left",
+  dropShadow: true,
+  dropShadowBlur: 10,
+  dropShadowDistance: 7,
+    fill: [
+    "red",
+    "#8800a3"
+  ],
+  fontFamily: "Times New Roman",
+  fontSize: 42,
+  wordWrap: true,
+  wordWrapWidth: 500,
+});
+// style for rule text
+const style1 = new PIXI.TextStyle({
+  align: "left",
+  dropShadow: true,
+  dropShadowBlur: 10,
+  dropShadowDistance: 7,
+    fill: [
+    "red",
+    "#d1ae00"
+  ],
+  fontFamily: "Times New Roman",
+  fontSize: 34,
+  wordWrap: true,
+  wordWrapWidth: 650,
+});
+
+let correctAnswer = null;
+
+//question text
+const questionText = new PIXI.Text('', style);
+questionText.x = 150;
+questionText.y = 75;
+app.stage.addChild(questionText);
+// confirmation text
+const confirmationText = new PIXI.Text('', style);
+confirmationText.x = 285;
+confirmationText.y = 350;
+app.stage.addChild(confirmationText);
+
+// text when player quits
+const quitText = new PIXI.Text('', style);
+quitText.x = 215;
+quitText.y = 250;
+app.stage.addChild(quitText);
+
+// hint player text
+const hintText = new PIXI.Text('', style1);
+hintText.x = 125;
+hintText.y = 350;
+app.stage.addChild(hintText);
+
+const pointsText = new PIXI.Text('', style)
+pointsText.x = 215;
+pointsText.y = 300;
+app.stage.addChild(pointsText);
+
+// to store player points
+var playerPoints = 0;
+
+
+
+
+// first question function
+// basic algebra question
+function question1(){
+  //Change the Welcome text to the question #
+  document.getElementById("demo").innerHTML = 'Question 1';
+  hintText.text = '';
+  pointsText.text = '';
+  playerPoints = 0;
+  quitText.text = '';
+  var rndmNum1 = Math.floor((Math.random() * 50) + 1);
+  var rndmNum2 = Math.floor((Math.random() * 1000) + 200);
+  questionText.text = (`Write an albebraic expression for the following verbal expression: The sum of ${rndmNum2} and ${rndmNum1} times a number(x).`);
+  //defines the correct answer for the question
+  correctAnswer = `${rndmNum1}x+${rndmNum2}`;
+}
+
+//question 2 function
+// square question
+function question2(){
+  document.getElementById("demo").innerHTML = 'Question 2';
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = '';
+  var rndmNum1 = Math.floor(Math.random() * (22 - 12 + 1)) + 12;
+  var power = 2;
+  var answr = Math.pow(rndmNum1, power)
+  questionText.text = `Which of the following values of x is a solution of the equation given: x^2 = ${answr}`;
+  correctAnswer = `${rndmNum1}`;
+}
+
+//question 3 function
+// distrutive property question[a(b+c)]
+function question3(){
+  document.getElementById('demo').innerHTML = "Question 3";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var rndmNum1 =  Math.floor(Math.random() * (20 - 2 + 1)) + 2;
```

Comment:
It looks like the intension is that these values are only set once... so they should be declared as constants `const` rather than `var`